src/sss_client/pam_sss.c

```diff
@@ -1846,6 +1868,12 @@ static int prompt_passkey(pam_handle_t *pamh, struct pam_items *pi,
     }
 
     kerberos_preauth = pi->passkey_key != NULL ? true : false;
+    if (!kerberos_preauth) {
+	    m[msg_idx].msg_style = PAM_TEXT_INFO;
```

Comment:
nitpicking - here you have wrong indentation (or is it just github ui?).

otherwise LGTM

Reply:
Thanks, fixed.